### DISCUSSION TITLE
Querying mapping tables

### DISCUSSION THREAD

**Discussion by multipliedtwice** [MichalLytek/typegraphql-prisma#374] [Q&A] [answered]
Hello, thank you for an excellent library. I'm stuck with one thing - I don't get generated FieldResolvers for mapping tables. I can query only IDs but can't query stuff related to that ID. Is there any workaround for that?

**Comment by MichalLytek** [comment]
1) Query for related stuff by id by having another graphql query
2) Use generated resolvers (relations resolvers)
3) Implement your own field resolvers for resolving relations

**Reply by multipliedtwice** [reply]
Thank you for an answer. Unfortunately, it's not getting generated for mapping tables. Maybe I need to enable some flag to make it work ?

**Reply by MichalLytek** [reply]
WDYM by mapping tables? Prisma uses implicit many-to-many join tables.

**Reply by multipliedtwice** [reply]
Yes, so I have a table like this

```
model companyOnUser {
  id        String      @id @default(uuid())
  companyId String
  company   company     @relation(fields: [companyId], references: [id], onDelete: Cascade)
  userId    String
  user      user        @relation(fields: [userId], references: [id], onDelete: Cascade)
  roleId    String
  role      companyRole @relation(fields: [roleId], references: [id], onDelete: Cascade)}
```

Let's say I'm trying to query companies with users. 
<img width="767" alt="image" src="https://user-images.githubusercontent.com/33694447/231085409-7a878dc5-e1eb-4270-a8f8-105a8aac1ae6.png">

I simply don't have an option to fill details on users, only select an ID

**Reply by MichalLytek** [reply]
Do you have loaded all the relation resolvers, also for your join tables? Or only the company and user ones?

**Reply by multipliedtwice** [reply, marked as answer]
@MichalLytek, you're right; I didn't include it! 
sorry for wasting your time. Once again - you've made a great library, thank you!
